packages/core/content-type-builder/admin/src/components/DataManagerProvider/index.js

```diff
@@ -20,7 +20,7 @@ import DataManagerContext from '../../contexts/DataManagerContext';
 import getTrad from '../../utils/getTrad';
 import makeUnique from '../../utils/makeUnique';
 import pluginId from '../../pluginId';
-// import FormModal from '../FormModal';
+import FormModal from '../FormModal';
```

Comment:
Can you keep this one commented for now, the view gets hard to navigate when we display it. I have tested your work and it seems to work correctly. Good job! 👏